setup.py

```diff
@@ -188,7 +188,7 @@ def run(self):
             'test': [
                 'flake8-docstrings>=1.5.0',
                 'flake8-quotes>=3.2.0',
-                'flake8>=3.8.4',
+                'flake8==5.0.4',
```

Comment:
`==` is not recommended.

Reply:
Changed to install with a min and max version range.